script/Arcueid/Day 7/07_00_ARC07_3_1_4.txt

```diff
@@ -188,7 +188,7 @@ I don't want to admit it, but...
 [sha:efcca97cf914663e2a4ab1f74b8abf38430eddfa]{
 -- Page 635, Offset 18177.
 -- 本当にお姫様だったんだな、こいつ……
-This woman truly is a princess...
+She really is a princess huh...
```

Comment:
Personal preference for more admiring/formal register here

Reply:
I prefer the new version personally.